services/app-web/src/pages/QuestionResponse/QATable/QuestionResponseRound.tsx

```diff
@@ -80,44 +42,11 @@ export const QuestionResponseRound = ({
                     </NavLinkWithLogging>
                 )}
             </div>
-            <table
-                className={`borderTopLinearGradient ${styles.qaDocumentTable}`}
-            >
-                <thead>
-                    <tr>
-                        <th scope="col">Document name</th>
-                        <th scope="col">Date added</th>
-                        <th scope="col">Added by</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    {documents.map((doc, index) => (
-                        <tr key={doc.name + index}>
-                            <td>
-                                {doc.downloadURL ? (
-                                    <LinkWithLogging
-                                        className={styles.inlineLink}
-                                        aria-label={`${doc.name} (opens in new window)`}
-                                        href={doc.downloadURL}
-                                        target="_blank"
-                                    >
-                                        {doc.name}
-                                    </LinkWithLogging>
-                                ) : (
-                                    doc.name
-                                )}
-                            </td>
-                            <td>
-                                {formatCalendarDate(
-                                    doc.createdAt,
-                                    'America/New_York'
-                                )}
-                            </td>
-                            <td>{getAddedByName(doc.addedBy)}</td>
-                        </tr>
-                    ))}
-                </tbody>
-            </table>
+          <QuestionDisplayTable
+            documents={extractDocumentsFromQuestion(question)}
```

Comment:
**Issue**: For both contract question and rate question responses, this function pulls the latest document to display in the response form. This works on the initial response to a question, but if you upload another response, the previous response is displayed instead of the question on the upload response page.

<img src="https://github.com/user-attachments/assets/bcb138b3-ec45-4c11-bd7e-a4a7459221d3" width="450" />